incubator/chartmuseum/templates/deployment-pvc.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.persistence.enabled -}}
```

Comment:
In the deployment it will choice between an existing claim or to generate a pvc.
In the PVC there needs to be a check if an existing claim is going to be used.

This can be done with the following if statement:
`{{- if not .Values.persistence.existingClaim -}}`

Reply:
Thanks for the review, seems that i missed that. Will consider it in future PRs.